.eslintrc.json

```diff
@@ -11,7 +11,6 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "prettier/@typescript-eslint",
```

Comment:
@poveden Is this no longer needed?

Reply:
No, it's not [since 8.0.0](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21).